develop/reference_ddl.rst

```diff
@@ -294,6 +294,60 @@ Not-null constraints can be applied to any column (distribution or not) because
 
   ALTER TABLE ads ALTER COLUMN image_url SET NOT NULL;
 
+Using NOT VALID Constraints
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some situations it can be useful to enforce constraints for new rows, while allowing existing non-conforming rows to remain unchanged. Citus supports this feature for CHECK constraints and foreign keys, using PostgreSQL's "NOT VALID" constraint designation.
```

Comment:
Maybe add a link to https://www.postgresql.org/docs/11/sql-altertable.html somewhere for further exploration of 'not valid'?